web/src/features/fba/components/infoPanel/InfoAccordion.tsx

```diff
@@ -20,24 +47,35 @@ const StyledAccordionSummary = styled(AccordionSummary)(() => ({
 }))
 
 // A component for rendering the provided title and content in an accordion format in the info panel.
-const InfoAccordion = ({ accordionDetailBackgroundColour, children, defaultExpanded, title }: InfoAccordionProps) => {
+const InfoAccordion = ({
+  accordionDetailBackgroundColour,
+  children,
+  defaultExpanded,
+  title,
+  advisoryStatus
+}: InfoAccordionProps) => {
   const theme = useTheme()
 
   return (
     <Accordion data-testid="info-accordion" disableGutters defaultExpanded={defaultExpanded} elevation={0}>
-      <StyledAccordionSummary expandIcon={<ExpandMoreIcon />}>
-        <Typography
-          data-testid="info-accordion-title"
-          sx={{
-            color: theme.palette.primary.main,
-            fontWeight: 'bold',
-            paddingLeft: '1.25rem'
-          }}
-          variant="h6"
-        >
-          {title}
-        </Typography>
-      </StyledAccordionSummary>
+      <Box>
+        <StyledAccordionSummary expandIcon={<ExpandMoreIcon />}>
+          <Typography
+            data-testid="info-accordion-title"
+            sx={{
+              color: theme.palette.primary.main,
+              fontWeight: 'bold',
+              paddingLeft: '1.25rem'
+            }}
+            variant="h6"
+          >
+            {title}
+          </Typography>
+        </StyledAccordionSummary>
+        {advisoryStatus && (
```

Comment:
I'm not a fan of the way the UI 'jumps' when switching between a fire zone unit that has an advisory/warning and one that doesn't. What do you think about removing this check for advisoryStatus and having the AdvisoryStatusBar display with the default background when there is no advisory status?

Reply:
Yep, can do something that. Will need to sort out some different styling logic in that case, otherwise every drop down will have a grey/white hash bar

Reply:
Yeah, definitely don't want a grey/white hashbar anywhere. I'm thinking just a solid bar all the time when there is no advisoryStatus.

Reply:
Addressed in 0b29a3b. Added `showAdvisoryStatusBar` so we don't also get an extra 10px grey bar on the stats dropdown, thoughts?

Reply:
Looks great!